docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,412 @@
+
+> **Good to know:**
+>
+> You may see the Router Cache being referred to as **Client-side Cache** or **Prefetch Cache**. While **Prefetch Cache** refers to the prefetched route segments, **Client-side Cache** refers to the whole Router cache, which includes both visited and prefetched segments.
+> This cache specifically applies to Next.js and Server Components, and is different to the browser's [bfcache](https://web.dev/bfcache/).
+
+#### Duration and Invalidation
+
+The cache is stored in the browser's temporary memory. Two factors determine how long the router cache lasts:
+
+- **Session**: The cache persists across [soft navigations](/docs/app/building-your-application/routing/linking-and-navigating#soft-navigation). However, it's cleared on page refresh or [hard navigation](/docs/app/building-your-application/routing/linking-and-navigating#hard-navigation).
+- **Automatic invalidation period**: The cache of an individual segment is automatically invalidated after a specific time. The duration depends on whether the route is [statically rendered](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#static-rendering-default) or [dynamically rendered](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering):
+  - **Dynamic routes**: 30 seconds
+  - **Static routes**: 5 minutes
+
+While a page refresh or hard navigation will clear **all** cached segments, the automatic invalidation period only affects the individual segment from the time it was last accessed or created.
+
+#### Opting out
+
+It's not possible to opt out of the Router Cache. However, you can opt out of prefetching by setting the `prefetch` prop of the `<Link>` component to `false`. The routes will still be cached when visited.
+
+## Cache Interactions
+
+When configuring the different caching mechanisms, it's important to understand how they interact with each other:
+
+### Data and Render
+
+- Invalidating or opting out of the Render Cache **does not** affect the Data Cache. This means you can dynamically render a route that has both cached and uncached data. This is useful when most of your page uses cached data (static), but you have a few components that rely on data that needs to be fetched at request time (dynamic). You can dynamically render without worrying about the performance impact of re-fetching all the data.
+- Revalidating the Data Cache **will** invalidate the Render Cache, as the render output depends on data.
+
+### Data and Router
+
+- Revalidating the Data Cache **will not** immediately invalidate the Router Cache due to the [automatic invalidation period](#duration-and-invalidation). This means the Router Cache will continue to serve the previous payload until a hard navigation/refresh, or the automatic invalidation period has elapsed.
```

Comment:
that is correct, unless they revalidate as part of an action

Reply:
Thank you!